example/test/react_test_components.dart

```diff
@@ -392,16 +392,6 @@ class _NewContextTypeConsumerComponent extends react.Component2 {
   }
 }
 
-var helloGregFunctionComponent = react.registerFunctionComponent(HelloGreg, displayName: 'HelloGreg');
-
-HelloGreg(Map props) {
```

Comment:
why you take away `HelloGreg`!?

Reply:
![3f98ri](https://user-images.githubusercontent.com/41018730/68153683-30c96300-ff03-11e9-968d-4cc785dfae9f.jpg)